run_service.sh

```diff
@@ -503,10 +503,12 @@ dotenv_set_key() {
 export_dotenv() {
     local dotenv_path="$1"
     unamestr=$(uname)
-    if [ "$unamestr" = 'Linux' ]; then
-        export $(grep -v '^#' $dotenv_path | xargs -d '\n')
-    elif [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
+    # Mac
+    if [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
```

Comment:
we only support darwin?

Reply:
https://megamorf.gitlab.io/2021/05/08/detect-operating-system-in-shell-script/

Seems uname or $OSTYPE would be similar.